oraclizeAPI_0.4.25.sol

```diff
@@ -303,23 +337,26 @@ contract usingOraclize {
     OraclizeAddrResolverI OAR;
 
     OraclizeI oraclize;
+    OraclizeIBytes oraclizeBytes;
+
     modifier oraclizeAPI {
         if((address(OAR)==0)||(getCodeSize(address(OAR))==0))
             oraclize_setNetwork(networkID_auto);
 
         if(address(oraclize) != OAR.getAddress())
             oraclize = OraclizeI(OAR.getAddress());
-
+            oraclizeBytes = OraclizeIBytes(OAR.getAddress());
```

Comment:
can save some ops by just doing `= address(oraclize)` here, no?

Reply:
Ahh wait yes, because the line before ensures `oraclize` is set. Good spot!